src/util/FetchUtil.ts

```diff
@@ -58,7 +58,7 @@ Promise<Representation> {
     logger.warn(`Missing content-type header from ${response.url}`);
     throw error;
   }
-  const contentTypeValue = parseContentType(contentType).type;
+  const contentTypeValue = parseContentType(contentType).value;
 
   // Try to convert to quads
   const representation = new BasicRepresentation(body, contentTypeValue);
```

Comment:
```suggestion
  // Try to convert to quads
  const representation = new BasicRepresentation(body);
  representation.metadata.contentType = contentType;
```
This way the representation also has the extra parameters should this be necessary.

Reply:
Agreed, but a `BasicRepresentation` constructor cannot only have  body (needs a contentType argument too).

Reply:
Ah right, good point. But that makes me think that perhaps we should just make it so we can provide the full content-type string there then. `BasicRepresentation` calls the `RepresentationMetadata` constructor here that takes a content-type string. So we probably want to also update the `RepresentationMetadata` constructor so that one supports full content-type inputs. 

Specifically the block here: https://github.com/solid/community-server/blob/e4f1e2aa0c068b3496f54593f2d060700142bed0/src/http/representation/RepresentationMetadata.ts#L89-L92
needs to be replaced by
```ts
 if (typeof overrides === 'string') { 
   this.contentType = overrides; 
 } else {
   this.setOverrides(overrides);
 }
```

And then you can have `const representation = new BasicRepresentation(body, contentType);` instead of just using the value.

This way the metadata constructor also has a broader content-type support without actually breaking anything anywhere.